maintainers/maintainer-list.nix

```diff
@@ -4574,6 +4574,11 @@
     githubId = 7875;
     name = "Rommel Martinez";
   };
+  eclairevoyant = {
+    github = "eclairevoyant";
+    githubId = 848000;
+    name = "éclairevoyant";
+  };
```

Comment:
could you add something like a email or matrix so someone would be able to send you a private message? there also is a quite active matrix space: https://matrix.to/#/#community:nixos.org

this isn't a blocker just a nice to have.

Reply:
I am currently in the process of figuring out self-hosting email, due to some issues with my current provider. Will add an email once that's sorted.